src/main/java/com/cloudblue/connect/api/models/CBCBy.java

```diff
@@ -0,0 +1,36 @@
+package com.cloudblue.connect.api.models;
+
+import com.cloudblue.connect.api.models.common.CBCEntity;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.mule.runtime.extension.api.annotation.param.Optional;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CBCBy implements CBCEntity {
```

Comment:
'By' is not an entity on connect please rename according to entity name like user or account whichever is suitable in your case. Also make sure you do not create duplicate entity in extension code.

I think you mean there as a connect user. Then it should be 'CBCUser'

Reply:
It could be confusing with the user collection. https://connect.cloudblue.com/community/api/openapi/#operations-Accounts-users_list_listAccountUsers